app/components/new-user.js

```diff
@@ -122,6 +127,17 @@ export default class NewUserComponent extends Component {
     });
   }
 
+  async validateUsernameCallback() {
+    const auths = await this.store.query('authentication', {
```

Comment:
Let's pre-load all authentication records at the top and then check the list. There aren't so many users that will break anything and the way it is built currently there is a new network request for every keystroke. Better to load the list when it is needed the first time and then use that list.